src/cloud/digitalocean/provisioner.ts

```diff
@@ -231,18 +207,6 @@ class Provisioner {
   }
 
   /**
-   * Dispatches status events
-   * events listed in STATUS_CODES
-   * @param {String} code one of STATUS_CODES 
-   */
-  private sendStatus_ = (code: string): void => {
-    this.dispatch_('status', {
```

Comment:
Nope! Sorry, forgot to mention that the UI doesn't listen for these since creating droplets is generally very fast, especially compared to installing uProxy.